aapcs64/aapcs64.rst

```diff
@@ -2026,6 +2026,21 @@ size\_t, ptrdiff\_t
 
 See `Arm C and C++ language mappings`_.
 
+Soft-float variant
+------------------
+
+For v8-R CPUs which do not have floating-point instructions or registers, the soft-float variant of the PCS can be used. Arm recommends that this is not used more widely, as most existing AArch64 implementations do have floating-point registers, and this variant is incompatible with the base variant.
+
+The soft-float variant is defined identically to the base variant, with these changes:
+
+* All floating-point types are passed as if they are an integer type with the same bit-width.
```

Comment:
Done, but I've dropped the word `signed` to avoid implying that the integer representation is meaningful, other then the bit-pattern matching the floating point value.